rss/rss09xparser.cpp

```diff
@@ -106,8 +106,12 @@ Item Rss09xParser::parse_item(xmlNode* itemNode)
 					start > 0 && authorfield[start] != '(';
 					start--) {
 				}
-				it.author = authorfield.substr(
-						start + 1, end - start);
+				if (start <= 0) {
```

Comment:
```suggestion
				if (start == 0) {
```

`unsinged int` can't be less than zero ;)